contrib/init/systemd/docker.service

```diff
@@ -9,7 +9,7 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd -H fd://
+ExecStart=/usr/bin/dockerd
```

Comment:
@thaJeztah what change do you think would be appropriate here?  Do you recognize the issue I'm raising and agree it's an issue?

It seems a bit of a chicken and egg scenario.  I want to use `daemon.json` but `-H fd://` is preventing me (although, I could add that to the hosts of the daemon.json).